### PR TITLE
chore: Improve docker and compose development

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
## Changes proposed

- Improve Docker and Docker Compose dev

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Pradumnasaraf** [comment]
Great catch Eddie. Updated the docs! Thank you!

File: README.md [hunk 1]
```diff
@@ -63,12 +63,16 @@ How friendly is your GitHub Open Source Repo? This project will check to make su
...
-      1. Start a new codespace
-      2. Run `npm ci` and `npm run dev`
-      3. When the project is running in the browser visit this and copy the generated URL. You will use this URL in place of localhost in step 8 above for creating your OAuth app
-      4. Update the environment file with your client id and secret.  Make sure to also update the `NEXTAUTH_URL=` to the generated codespace URL as well.
-      5. Continue with step `9b` to run the docker container and database migration.
+      4. Start a new codespace
+      5. Run `npm ci` and `npm run dev`
+      6. When the project is running in the browser visit this and copy the generated URL. You will use this URL in place of localhost in step 8 above for creating your OAuth app
+      7. Update the environment file with your client id and secret.  Make sure to also update the `NEXTAUTH_URL=` to the generated codespace URL as well.
+      8. Continue with step `9b` to run the docker container and database migration.
```
Comment (eddiejaoude) [inline README.md:77]: I think these should start from `1` again because it is under a new section `c`

**Comment by eddiejaoude** [comment]
Looks good, just a small comment on the number

File: README.md (hunk 1 above)
Comment (eddiejaoude) [inline README.md:77]:
```suggestion
      1. Start a new codespace
      2. Run `npm ci` and `npm run dev`
      3. When the project is running in the browser visit this and copy the generated URL. You will use this URL in place of localhost in step 8 above for creating your OAuth app
      4. Update the environment file with your client id and secret.  Make sure to also update the `NEXTAUTH_URL=` to the generated codespace URL as well.
      5. Continue with step `9b` to run the docker container and database migration.
```

File: README.md [hunk 2]
```diff
@@ -63,7 +63,11 @@ How friendly is your GitHub Open Source Repo? This project will check to make su
 
    a. If you have Postgres installed, you can run the app locally `npm run dev` OR
    
-   b. If you don't have Postgres installed, you can use Docker Compose to run the database and the app `docker compose -f docker-compose.yaml up -d`, after the containers are ready, run the command `npm run db:migrate:dev`
+   b. Running with Docker Compose (Recommended if you don't have Postgres installed)
+      1. Once you done above steps and have the `.env` file ready
+      1. Run the command `docker compose up -d`
+      2. Once the containers are ready, run the command `npm run db:migrate:dev`
+      3. Visit `http://localhost:3000` in your browser
```
Comment (eddiejaoude) [inline README.md:70]: the number `1` appears twice

```suggestion
      1. Once you done above steps and have the `.env` file ready
      2. Run the command `docker compose up -d`
      3. Once the containers are ready, run the command `npm run db:migrate:dev`
      4. Visit `http://localhost:3000` in your browser
```

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by Pradumnasaraf** [comment]
> Looks good, just a small comment on the number

My bad Eddie. Thank you for fixing

**Pull request merged**